app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/itemsList/itemsListHeader/itemsListHeader.jsx

```diff
@@ -34,19 +34,28 @@ export const ItemsListHeader = ({
   showErrorLogs,
   onShowErrorLogsChange,
   optionValue,
+  isShownLess,
+  isBulkOperation,
 }) => {
   const { formatMessage } = useIntl();
   const [isAllSelected, setIsAllSelected] = useState(selectedItems.length === testItems.length);
   useEffect(() => {
     setIsAllSelected(selectedItems.length === testItems.length);
   }, [selectedItems]);
 
+  const setSelectedItems = () => {
```

Comment:
There is no need to carry this logic into a separate function, it can be done directly in `onCheckboxChange`.